### PR TITLE
Improve accessible name for images on home page and prop names for HomeSection

### PR DESCRIPTION
**Pull request by milofultz**
## This PR:

Improves accessible name for images on home page.
 
**1.**  Adds context for all logo images
**2.** Removes unnecessary `aria-label`s
**3.** Mark decorative (non-informative images) as such using an empty `alt` attribute

### Components

```
src/components/Footer/RenderCompanyInfoSection.jsx
src/components/NavBar/NavbarDesktop.jsx
src/components/NavBar/NavbarLoggedOut.jsx
src/components/NavBar/NavbarMobile.jsx
src/pages/Home.jsx
```

### Tests

```
test/components/NavBar/NavbarDesktop.test.jsx
test/components/NavBar/NavbarLoggedOut.test.jsx
test/components/NavBar/NavbarMobile.test.jsx
test/pages/__snapshots__/Home.test.jsx.snap
```

## Screenshots (if applicable):

Should be no difference visually.

## Additional Context (optional):

Following guidance around [decorative](https://www.w3.org/WAI/tutorials/images/decorative/#example-1-image-used-as-part-of-page-design)/[informative](https://www.w3.org/WAI/tutorials/images/informative/) images from W3. Verified using [this extension](https://accessibilityinsights.io/) and DevTools.

### CODE REVIEW

File: src/components/NavBar/NavbarDesktop.jsx
Comment (leekahung) [inline src/components/NavBar/NavbarDesktop.jsx]: The png in question is the PASS specific logo. The Code PDX has a different look.

File: src/components/NavBar/NavbarLoggedOut.jsx
Comment (leekahung) [inline src/components/NavBar/NavbarLoggedOut.jsx]: See comment above about logo.

File: src/components/NavBar/NavbarMobile.jsx
Comment (leekahung) [inline src/components/NavBar/NavbarMobile.jsx]: See logo comment above.

File: src/pages/Home.jsx
Comment (leekahung) [inline src/pages/Home.jsx]: It seems like this is a prop used to pass in the description for the image alt encased in the component. Think we could rename this to componentImgAlt or something. Same with src.

File: src/pages/Home.jsx
Comment (leekahung) [inline src/pages/Home.jsx]: See comment about alt above.

File: src/pages/Home.jsx
Comment (leekahung) [inline src/pages/Home.jsx]: See comment about alt above.

**Review by leekahung — CHANGES_REQUESTED** [review]
Mostly about image `alt` tags. See feedback. Think it's good to go after they're addressed.

File: src/components/NavBar/NavbarDesktop.jsx
Comment (milofultz) [reply src/components/NavBar/NavbarDesktop.jsx]: Ah perfect, will fix 👍

File: src/pages/Home.jsx
Comment (milofultz) [reply src/pages/Home.jsx]: Yeah good point. I'll get that.

File: src/components/Home/HomeSection.jsx
Comment (milofultz) [inline src/components/Home/HomeSection.jsx:10]: @leekahung mentioned that `alt` should be more aptly named since it's only for the component image. Since `src` was the same, I thought I should lump it in.

**Review by xscottxbrownx — APPROVED** [review]
I read the html spec to clarify the empty string given to `alt` and seems legit due to the surrounding text.

I verified the regex in the tests for the changes to the pass logo.

All changes seem legit to me - not sure the title of PR is accurate anymore though 😅

**Comment by xscottxbrownx** [comment]
@leekahung I believe this is ready for you to approve now - so merging is possible.

**Review by leekahung — COMMENTED** [review]
Yeah, I'm mostly fine with this. Although I've notice the image `alts` for `HomeSection` are all empty now compared to before. Won't it be necessary for accessibility?

https://accessibility.psu.edu/images/imageshtml/

**Comment by milofultz** [comment]
> Yeah, I'm mostly fine with this. Although I've notice the image `alts` for `HomeSection` are all empty now compared to before. Won't it be necessary for accessibility?
> 
> https://accessibility.psu.edu/images/imageshtml/

I went with the info about [decorative](https://www.w3.org/WAI/tutorials/images/decorative/#example-1-image-used-as-part-of-page-design)/[informative](https://www.w3.org/WAI/tutorials/images/informative/) images from W3 (see example 4 in decorative as that's what I thought it fell under). It was a judgment call that I decided they were decorative since I'm not sure they contribute to the content. Happy to hear otherwise, it's a tough call.

**Comment by leekahung** [comment]
> I went with the info about [decorative](https://www.w3.org/WAI/tutorials/images/decorative/#example-1-image-used-as-part-of-page-design)/[informative](https://www.w3.org/WAI/tutorials/images/informative/) images from W3 (see example 4 in decorative as that's what I thought it fell under). It was a judgment call that I decided they were decorative since I'm not sure they contribute to the content. Happy to hear otherwise, it's a tough call.

Ah, I see. Well, considering that the section title follows immediately after the images, I think it should be fine even if the image themselves break. Alright, I'll approve this.

**Review by leekahung — APPROVED** [review]

**Comment by leekahung** [comment]
Hey @milofultz. Was planning to merge this branch in after resolving a merge conflict for one of the test files. 

Unfortunately, the resolution I've attempted to make seemed to have failed the test. I think you'll be able to fix it from your end (sorry for the inconvinence). If the test gets fixed, let me know, I'll have this merged into `Development`.

Thanks!

**Comment by milofultz** [comment]
@leekahung Should be good to go now 👍

**Pull request merged**
